### DISCUSSION TITLE
Importing a Vanjaro Template to another Vanjaro Site?

### DISCUSSION THREAD

**Discussion by joshgk00** [vanjarosoftware/Vanjaro.Platform#890] [Q&A] [answered]
I created a page and saved it as a Page Template. I exported the page template and it created a zip file. There are a couple of assets that were included in the template export as well.  Is there a way to import that exported page template to another Varjaro site?

**Comment by meetmandeep** [comment, marked as answer]
It's not possible to import a template at this time. The export functionality was created to support Vanjaro Template Library. The code exists in framework to import but there's no UI at the moment. I would recommend creating an issue so it can be worked on in future. 

You can easily save a page as a template for Site > Pages

You can also move this template to another Vanjaro Site  by copying the .json file from /Portals/PortalID/vThemes/templates/pages to the new site. You'll have to manually move any referenced assets such as images or update any existing links within template.

**Reply by Tychodewaard** [reply]
https://github.com/vanjarosoftware/Vanjaro.Platform/issues/1582 issue created
